src/sagas/index.js

```diff
@@ -835,7 +836,15 @@ function* fetchAllTemplates (action) {
   const templates = yield callExternalAction('getAllTemplates', Api.getAllTemplates, action)
 
   if (templates && templates['template']) {
-    const templatesInternal = templates.template.map(template => Api.templateToInternal({ template }))
+    const templatesInternal = (yield all(
+      templates.template
+        .map(function* (template) {
+          const templateInternal = Api.templateToInternal({ template })
+          templateInternal.permits = yield fetchPermits({ entityType: PermissionsType.TEMPLATE_TYPE, id: template.id })
```

Comment:
I hate that we have to do this.  Maybe a better RFE to writeup would be to have the API be able to directly give us a user's permits on an entity instead of having to do through the permissions and roles?

@sgratch , @michalskrivanek - Do you think an approach like that wold be more likely to be implemented over a three level follow as the [current RFE BZ1639784](https://bugzilla.redhat.com/show_bug.cgi?id=1639784) asks for?